nextflow/configs/container.config

```diff
@@ -22,7 +22,7 @@ process {
 
     withLabel: plot_contig_map {container = 'microbiomeinformatics/emg-viral-pipeline-plot-contig-map:v1' }
     withLabel: virfinder {      container = 'microbiomeinformatics/virfinder:v1.1__eb8032e' }
-    withLabel: checkV {         container = 'microbiomeinformatics/checkv:v0.8.1' }
+    withLabel: checkV {         container = 'microbiomeinformatics/checkv:v0.8.1__1' }
```

Comment:
just a recommendation: for such cases you could also, in general, add an md5 checksum to the container name. E.g. the first 5 characters of the md5 sum of the `Dockerfile`.

Reply:
Captured in https://github.com/EBI-Metagenomics/emg-viral-pipeline/issues/85